docs/common/craft-parts/reference/plugins/rust_plugin.rst

```diff
@@ -174,8 +174,22 @@ To get even better performance, you might want to follow the tips below.
 
 * (Advanced) Perform cross-language LTO. This requires installing the correct version of LLVM/Clang and setting the right environment variables.
   You must know which LLVM version of your selected Rust toolchain is using.
-  For example, Rust 1.71 uses LLVM 16 because you can see it bundles a ``libLLVM-16-rust-1.71.1-stable.so`` file in the ``lib`` directory.
-  In this case, you want to install ``clang-16`` and ``lld-16`` from the Ubuntu archive.
+  You can use ``rustc -vV`` to print out the LLVM version used by the compiler. For example, you can see Rust 1.81 uses LLVM 18.1 because
+  it prints out an output like this:
+
+  .. code-block:: text
+    rustc 1.81.0 (eeb90cda1 2024-09-04)
+    binary: rustc
+    commit-hash: eeb90cda1969383f56a2637cbd3037bdf598841c
+    commit-date: 2024-09-04
+    host: x86_64-unknown-linux-gnu
+    release: 1.81.0
+    LLVM version: 18.1.7
+
+  On Rust toolchains that do not include the LLVM version, you can still check the LLVM version number by examining the ``lib`` directory.
+  For example, Rust 1.81 uses LLVM 18.1 because it bundles a ``libLLVM.so.18.1-rust-1.81.0-stable`` file under the ``lib`` directory.
+
+  In this case, you want to install ``clang-18`` and ``lld-18`` from the Ubuntu archive.
```

Comment:
```suggestion
  For example, Rust 1.81 uses LLVM 18.1 because it bundles a ``libLLVM.so.18.1-rust-1.81.0-stable`` file under the ``lib`` directory.
  In this case, you would install ``clang-18`` and ``lld-18`` from the Ubuntu archive.
```